### DISCUSSION TITLE
Q-Editor add table

### DISCUSSION THREAD

**Discussion by LadIQe** [quasarframework/quasar#10759] [General - Components / Directives / etc]
Hello,

do someone know how to add table as toolbar button for Q-Editor? There is cmd property, but I have no idea how it works, there is nothing in docs about how to write it properly.

Thanks

**Comment by MilosPaunovic** [comment]
You can start here, with the example of adding new commands to toolbar.
https://quasar.dev/vue-components/editor#example--add-new-commands

**Reply by LadIQe** [reply]
Hi, thank you for reply.

Yes I was looking into it, but as I said, I had no luck to get it done. I have problem with "cmd" property for definitions, because that property for execCommand method. Handler looks ideal for void functions, like download

But feel free to correct me if Im wrong
